.github/CONTRIBUTING.md

```diff
@@ -34,11 +34,11 @@ This package contains Composer scripts to quickly run the developer checks which
 
 After `composer install`, you can do:
 
- - `composer test`: **Run all checks and tests** - this should pass cleanly before you submit a pull request.
+ - `composer check`: **Run all checks and tests** - this should pass cleanly before you submit a pull request.
      - `composer lint`: Just run PHP and XML linters.
-     - `composer phpcs`: Just run PHPCS against this package.
-     - `composer phpunit`: Just run the unit tests.
-     - `composer ruleset`: Just run the ruleset tests.
+     - `composer cs`: Just run PHPCS against this package.
+     - `composer test`: Just run the unit tests.
+     - `composer test-ruleset`: Just run the ruleset tests.
```

Comment:
While you are at it, what about removing the "just" from those phrases ?

Reply:
Context: "just" can be seen as ablist language and doesn't add much value to the phrasing. If so preferred, it could be interchanged for "Only", but IMO it could just as well be dropped.

Reply:
Agreed - updated to use the same descriptions as in `composer.json`.